### DISCUSSION TITLE
Notification closes right after input on widget - Keep open?

### DISCUSSION THREAD

**Discussion by carorm** [fabiusBile/Spotify4BigSur#5] [Ideas]
Would be nicer UX to persist the widget and not have it close after user input and only close if the user clicks away from the notifications center. Similar to Slack where clicking on the individual messages will expand but not necessarily close out the notification until clicking on it. Especially helpful if user wants to skip ahead or reverse back more than once.

**Comment by fabiusBile** [comment]
Unfortunately it's impossible - that's how all Notification Center widgets work in Big Sur. I hope Apple will change this.
